examples/alerts/alerts.md

```diff
@@ -0,0 +1,143 @@
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+- alert: ThanosCompactBucketOperationsFailed
+  expr: rate(thanos_objstore_bucket_operation_failures_total{app="thanos-compact"}[5m]) > 0
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Compact bucket operations are failing
+    impact: Long term storage queries will be slower 
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: COMPACTION_URL
+```
+
+## Ruler
+
+For Thanos ruler we run alerts some alerts in local Prometheus:
+
+```
+- alert: ThanosRuleIsDown
+  expr: up{app="thanos-rule"} == 0 or absent(up{app="thanos-rule"})
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Rule is down
+    impact: Alerts are not working
+    action: 'check {{ $labels.kubernetes_pod_name }} pod in {{ $labels.kubernetes_namespace}} namespace'
+    dashboard: RULE_DASHBOARD
+- alert: ThanosRuleIsDroppingAlerts
+  expr: rate(thanos_alert_queue_alerts_dropped_total{app="thanos-rule"}[5m]) > 0
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Rule is dropping alerts
+    impact: Alerts are not working
+    action: 'check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace'
+    dashboard: RULE_DASHBOARD
+```
+
+And some alerts running in thanos rule:
+
+```
+- alert: ThanosRuleGrpcErrorRate
+  expr: rate(grpc_server_handled_total{grpc_code=~"Unknown|ResourceExhausted|Internal|Unavailable",app="thanos-rule"}[5m]) > 0
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Rule is returning Internal/Unavailable errors
+    impact: Recording Rules are not working
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: RULE_DASHBOARD
+```
+
+## Store Gateway
+
+```
+- alert: ThanosStoreGrpcErrorRate
+  expr: rate(grpc_server_handled_total{grpc_code=~"Unknown|ResourceExhausted|Internal|Unavailable",app="thanos-store"}[5m]) > 0
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Store is returning Internal/Unavailable errors 
+    impact: Long Term Storage Prometheus queries are failing
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: GATEWAY_URL
+```
+
+## Sidecar
+
+```
+- alert: ThanosSidecarPrometheusDown
+  expr: thanos_sidecar_prometheus_up{name="prometheus"} == 0
+  for: 5m
+  labels:
+    team: TEAM
+  annotations:
+    summary: Thanos Sidecar cannot connect to Prometheus
+    impact: Prometheus configuration is not being refreshed
+    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
+    dashboard: SIDECAR_URL
+- alert: ThanosSidecarBucketOperationsFailed
```

Comment:
maybe similar for store?

Reply:
Added in https://github.com/improbable-eng/thanos/pull/392/files#diff-fc17f1cae7faf239396714ad90fd3d06R118 :)